atom/browser/api/event_emitter.h

```diff
@@ -51,25 +51,25 @@ class EventEmitter : public Wrappable<T> {
   template <typename... Args>
   bool EmitCustomEvent(const base::StringPiece& name,
                        v8::Local<v8::Object> event,
-                       const Args&... args) {
+                       Args&&... args) {
```

Comment:
Does this mean that emit arguments should always be movable ?

Reply:
For template args, this means _universal reference_ (or _forward reference_):
https://isocpp.org/blog/2012/11/universal-references-in-c11-scott-meyers